### PR TITLE
 Fix broken link in NovoGrad docstring

### LINKED ISSUE #2085: Markdown parsing error in NovoGrad documentation page
The documentation page of NovoGrad has a broken Markdown link shown below:

novograd

I suspect this is because only the first line of the docstring is used for the first line of the documentation page. In the docstring (shown below), the link spans multiple lines.

https://github.com/tensorflow/addons/blob/5f746971d0d9491716f2f13206299a2c45941b0c/tensorflow_addons/optimizers/novograd.py#L26-L28

The simplest solution would be to edit the docstring so that there is a one-line summary for the first line with a paper link after the newline. One example would be [LAMB's documentation page](https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers/LAMB). I will create a PR with this solution.

### PR DESCRIPTION
**Pull request by seungjaeryanlee**
# Description

This PR fixes stylistic issues in the [NovoGrad optimizer documentation page](https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers/NovoGrad).

Fixes #2085

## Type of change

- [ ] Bug fix
- [ ] New Tutorial
- [x] Updated or additional documentation
- [ ] Additional Testing
- [ ] New Activation and the changes conform to the [activation contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/activations/README.md#contribution-guidelines)
- [ ] New Callback and the changes conform to the [callback contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/callbacks/README.md#contribution-guidelines)
- [ ] New Image addition and the changes conform to the [image op contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/image/README.md#contribution-guidelines)
- [ ] New Layer and the changes conform to the [layer contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/README.md#contribution-guidelines)
- [ ] New Loss and the changes conform to the [loss contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/losses/README.md#contribution-guidelines)
- [ ] New Metric and the changes conform to the [metric contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/metrics/README.md#contribution-guidelines)
- [ ] New Optimizer and the changes conform to the [optimizer contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/optimizers/README.md#contribution-guidelines)
- [ ] New RNN Cell and the changes conform to the [rnn contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/rnn/README.md#contribution-guidelines)
- [ ] New Seq2seq addition and the changes conform to the [seq2seq contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/seq2seq/README.md#contribution-guidelines)
- [ ] New Text addition and the changes conform to the [text op contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/text/README.md#contribution-guidelines)

# Checklist:

- [x] I've properly [formatted my code according to the guidelines](https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#coding-style)
    - [x] By running Black + Flake8
    - [x] By running pre-commit hooks
- [x] This PR addresses an already submitted issue for TensorFlow Addons
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] This PR contains modifications to C++ custom-ops

# How Has This Been Tested?

If you're adding a bugfix or new feature please describe the tests that you ran to verify your changes:
*

### CODE REVIEW

File: tensorflow_addons/optimizers/novograd.py [hunk 1]
```diff
@@ -23,16 +23,17 @@
...
-    stable. More specifically we compute (for more information on the
+    """Optimizer that implements NovoGrad.
+
+    The NovoGrad Optimizer was first proposed in [Stochastic Gradient
+    Methods with Layerwise Adaptive Moments for training of Deep
+    Networks](https://arxiv.org/pdf/1905.11286.pdf) NovoGrad is a
+    first-order SGD-based algorithm, which computes second moments per
+    layer instead of per weight as in Adam. Compared to Adam, NovoGrad
+    takes less memory, and has been found to be more numerically stable.
+    More specifically we compute (for more information on the
```
Comment (WindQAQ) [inline tensorflow_addons/optimizers/novograd.py:34]: Let's remove the sentence `More specifically we compute`.

Comment (seungjaeryanlee) [reply tensorflow_addons/optimizers/novograd.py]: Done!
